common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -128,7 +128,7 @@ protected[core] case class BlackBoxExec(override val image: String, code: Option
 }
 
 protected[core] case class SequenceExec(components: Vector[FullyQualifiedEntityName]) extends Exec(Exec.SEQUENCE) {
-    override def size = components.map(c => c.size).reduce(_ + _)
+    override def size = components.map(c => c.size).reduceOption(_ + _) getOrElse (0 B)
```

Comment:
@markusthoemmes once we limit the length of the action names, i'm ok turning this into a no-op (return `0.B`).